tutorial/Linux_Tutorial.md

```diff
@@ -386,3 +386,34 @@ $ python3 safe_zone_publisher.py
 ```
 
 Here it fails with `rt/cmd_vel topic not found in allow rule.` meaning that the not is not allowed to publish on the ROS topic `/cmd_vel`.
+
+
+##### Simulating the robot in Gazebo
+
+Make sure to kill all the running nodes.
```

Comment:
great opening :)

Reply:
![sup](https://user-images.githubusercontent.com/6644208/59035834-6abc0d80-886e-11e9-93fa-484228365454.gif)

Reply:
nothing like a  grand entrance for a tutorial :)